eng/common/instructions/azsdk-tools/typespec-to-sdk.instructions.md

```diff
@@ -57,10 +57,12 @@ Wait for the user to respond with a confirmation before proceeding to Step 1. Us
 **Goal**: Determine how to generate SDKs
 **Actions**:
 1. Present options: "How would you like to generate SDKs?"
-    - Option A: "Generate SDK locally". This is currently supported only for Python. Do not recommend this for other languages.
+    - Option A: "Generate SDK locally".
     - Option B: "Use SDK generation pipeline"
 2. Based on selection:
-    - If Option A: Refer to #file:create-sdk-locally.instructions.md and then proceed to Step 6
+    - If Option A: 
+        - Follow #file:./local-sdk-workflow.instructions.md to generate and compile the SDK.
+        - After SDK has been generated, to continue the SDK release, users can create the SDK pull request manually then proceed to Step 9.
```

Comment:
it might be better that the PR is not created here because the SDK is still missing configurations, and a release plan to make it ready for release.
continuing to step 9 should be enough

Reply:
The following steps after step9 will help with release plan creation. I'm not quite sure what missing configurations you refer to. But I think it doesn't block the PR creation and missing pieces should be caught in either CI or release plan.